Cargo.toml

```diff
@@ -58,7 +59,7 @@ percent-encoding = { version = "2", optional = true }
 rand = "0.7"
 redis = { version = "0.17", optional = true }
 regex = "1"
-reqwest = { version = "0.9.11", optional = true }
+reqwest = { version = "0.10", optional = true, features = ["blocking", "json"] }
```

Comment:
Can we upgrade to `v0.11`? If not, why not go for `v0.10.10`?

Reply:
Presumably because 0.11 is 5 days old. (v0.10.10 *is* what's being targeted here, see Cargo.lock)

Note, though, that the dist tests are busted.

Reply:
> Can we upgrade to v0.11? If not, why not go for v0.10.10?

Yes, but I hesitated to do it by reasons:

1. what @glandium said.
2. I'm not sure that it's nice to work, so I intend to avoid to include (at least) 3 versions Tokio in our dependency graph, v0.1 come from direct dependency, v0.2 come from tokio-compat, v1.0 comes from reqwest 0.11.....

-------

> Note, though, that the dist tests are busted.

Ugh, but I don't have any Linux environment to test failure cases...

-----

I'll address the conflict and push it once.

Reply:
> Presumably because 0.11 is 5 days old. (v0.10.10 is what's being targeted here, see Cargo.lock)

Yes, indeed. My mistake. :)